### DISCUSSION TITLE
Botpress application integration

### DISCUSSION THREAD

**Discussion by johackim** [chatwoot/chatwoot#8460] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

Integrate an open-source bot to auto-reply messages : https://github.com/botpress/botpress/

**Describe the solution you'd like**

When a message is sended to chatwoot via the livechat, botpress receive the message and automatically reply to the end-user.

**Comment by iammattmartin** [comment]
This looks really good.

**Comment by valentinwinkelmann** [comment]
That would be super cool as an integration!

**Comment by mkbadri** [comment]
+1 definitely a value add, will be a really good combination.

**Comment by sanjayws** [comment]
I made a very simple yet workable Botpress integration with CW using the great guide from here https://github.com/chatwoot/rasa-agent-bot-demo. Do reach out, happy to share my very rudimentary php code.

**Comment by sojan-official** [comment]
@sanjayws  if you have the code in a git repo, please feel free to share it. I am sure the community will be happy to play around with it and give feedback.

**Comment by rodenp** [comment]
Is there any progress on this integration?

**Comment by sojan-official** [comment]
@rodenp it's still in our backlog.  The team isn't actively working on this at the moment.

**Comment by sojan-official** [comment]
please follow some of the community integrations:

https://github.com/chatwoot/implementation-examples/tree/main/chatwoot-botpress-bridge
https://github.com/weunlearn/Chatwoot-Botpress-Bridge
https://github.com/douglara/chatwoot-botpress-bridge

**Comment by WevrLabs** [comment]
is this on the roadmap for the near future?
@sojan-official @pranavrajs

**Comment by pranavrajs** [comment]
I'm moving this to the discussions as we will not be prioritizing this immediately.
